oeo-physical.omn

```diff
@@ -1335,7 +1345,8 @@ Class: Geothermal
         <http://purl.obolibrary.org/obo/IAO_0000119> "REGULATION (EC) No 1099/2008 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 22 October 2008 on energy statistics https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32008R1099&from=EN"@en
     
     SubClassOf: 
-        Heat
+        Heat,
```

Comment:
The class should be called 'GeothermalHeat' or 'GeothermalEnergy' instead of just 'geothermal'

Reply:
I'll change that